src/views/Timesheets/components/SavedShiftsList/index.jsx

```diff
@@ -175,7 +178,7 @@ export default class SavedShiftsList extends Component {
     );
 
     let header = (
-      <Grid item xs={12} style={styles2.headerStyle}>
+      <Grid item xs={12} style={styles2.headerStyle} className={styles.timesheets_headerStyle}>
```

Comment:
Same thing here, should move everything from styles2 into the CSS class you created